moskito-backend/src/Entity/CampsiteFeature.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\CampsiteFeatureRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=CampsiteFeatureRepository::class)
+ */
+class CampsiteFeature
+{
+
+    const TYPE_WLAN = "kein WLAN";
```

Comment:
würde hier TYPE_WLAN = 'wlan' nur definieren
Backend ist am besten sprachneutral, so dass "Übersetzungen" im Frontend passieren

und im Frontend dieses Mapping machen 'wlan' => 'kein WLAN'

so kann man Texte direkt im Frontend ändern ohneim BE deployen zu müssen.